docker/community/docker-compose.yml

```diff
@@ -186,70 +171,14 @@ services:
       REACT_APP_WIDGET_EMBED_PATH: ${WIDGET_EMBED_PATH}
       REACT_APP_IS_SELF_HOSTED: 'true'
       REACT_APP_WS_URL: ${REACT_APP_WS_URL}
-      IS_MULTI_TENANCY_ENABLED: ${IS_MULTI_TENANCY_ENABLED}
     ports:
       - 4200:4200
     healthcheck:
-      test: ["CMD-SHELL", "wget --spider --quiet http://localhost:4200 || exit 1"]
-      interval: 30s        
-      timeout: 10s        
-      retries: 3       
-      start_period: 10s    
-
-  widget:
-    image: 'ghcr.io/novuhq/novu/widget:0.24.0'
-    depends_on:
-      api:
-        condition: service_healthy
-      worker:
-        condition: service_healthy
-      web:
-        condition: service_healthy
-    container_name: widget
-    restart: unless-stopped
-    logging:
-      driver: 'json-file'
-      options:
-        max-size: '50m'
-        max-file: '5'
-    environment:
-      REACT_APP_API_URL: ${API_ROOT_URL}
-      REACT_APP_WS_URL: ${REACT_APP_WS_URL}
-      REACT_APP_ENVIRONMENT: ${NODE_ENV}
-      WIDGET_CONTEXT_PATH: ${WIDGET_CONTEXT_PATH}
-    ports:
-      - 4500:4500
-    healthcheck:
-      test: ["CMD-SHELL", "wget --spider --quiet http://localhost:4500 || exit 1"]
-      interval: 30s      
-      timeout: 10s       
-      retries: 3         
-      start_period: 10s   
-
-  embed:
-    image: 'ghcr.io/novuhq/novu/embed:0.24.0'
-    depends_on:
-      widget:
-        condition: service_healthy
-    container_name: embed
-    restart: unless-stopped
-    logging:
-      driver: 'json-file'
-      options:
-        max-size: '50m'
-        max-file: '5'
-    environment:
-      WIDGET_URL: ${WIDGET_URL}
-      NEW_RELIC_APP_NAME: ${NEW_RELIC_APP_NAME}
-      NEW_RELIC_LICENSE_KEY: ${NEW_RELIC_LICENSE_KEY}
-    ports:
-      - 4701:4701
-    healthcheck:
-      test: ["CMD-SHELL", "wget --spider --quiet http://localhost:4701 || exit 1"]
-      interval: 30s       
-      timeout: 10s        
-      retries: 3          
-      start_period: 10s    
+      test: ['CMD-SHELL', 'curl --silent --fail http://localhost:4200 || exit 1'] # Changed from wget to curl
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** I think we should remove the comment as it loses context after this change‏
```suggestion
      test: ['CMD-SHELL', 'curl --silent --fail http://localhost:4200 || exit 1']
```

Reply:
Sure